build.gradle

```diff
@@ -13,7 +13,13 @@ repositories {
 }
 
 dependencies {
-	implementation 'org.springframework.boot:spring-boot-starter-data-rest'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+
+	implementation 'mysql:mysql-connector-java'
+	compile group: 'org.springframework.boot', name: 'spring-boot-starter-thymeleaf'
+    compileOnly 'org.projectlombok:lombok:1.18.10'
+    annotationProcessor 'org.projectlombok:lombok:1.18.10'
+
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 }
```

Comment:
Do not send a pull request to the master branch, you can send it to develop branch. And also make sure you updated with the latest code changes.